sdk/storage/storage-blob/package.json

```diff
@@ -37,21 +37,19 @@
     "execute:samples": "npm run build:samples && npm run execute:js-samples && npm run execute:ts-samples",
     "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "mocha --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --full-trace -t 120000 --retries 2 dist-test/index.node.js",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --full-trace -t 120000 --retries 2 dist-esm/test/*.spec.js dist-esm/test/node/*.spec.js",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint -c ../../.eslintrc.json src test samples --ext .ts --fix",
     "lint": "eslint -c ../../.eslintrc.json src test samples --ext .ts -f html -o storage-blob-lintReport.html || exit 0",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
-    "test:browser": "npm run clean && npm run build:test && npm run integration-test:browser",
-    "test:node": "npm run clean && npm run build:test && npm run integration-test:node",
-    "test": "npm run clean && npm run build:test && npm run integration-test",
-    "unit-test:browser": "cross-env TEST_MODE=playback npm run integration-test:browser",
-    "unit-test:node": "cross-env TEST_MODE=playback npm run integration-test:node",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test:browser": "cross-env TEST_MODE=playback karma start --single-run",
+    "unit-test:node": "cross-env TEST_MODE=playback mocha --require source-map-support/register --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=- --full-trace -t 120000 --retries 2 dist-test/index.node.js",
```

Comment:
One minor concern is duplication. Otherwise looks good.

Reply:
Though we were previously having nyc in the `integration-test` scripts, we were not capturing the code coverage reports in the CI.

Me and Jeremy, we were thinking of having `coverage-integration-test` scripts for the CI instead of `integration-test` script, also get the code coverage reports from it in the CI for the nightly builds.

Reply:
Your update seems fine, but we need to capture the code coverage reports in the CI nightly tests too.

Reply:
Yes, #6624 should take care of capturing the reports and presenting the results.
My main concern was to not require a separate npm script for it.

Regarding duplication, it was already present as mocha was being run slightly differently for live test vs nyc.

Reply:
Fine, thanks!